.github/workflows/format-and-commit.yml

```diff
@@ -0,0 +1,39 @@
+name: Format And Commit
+
+concurrency:
+  group: ${{ github.ref }}
+
+on:
+    # Pending getting through individual PRs.
+    # push:
+    #   branches:
+    #     - master
+    workflow_dispatch: ~
+
+permissions:
+  contents: read
+
+jobs:
+  dprint-fmt:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        check-latest: true
+        node-version: '*'
+    - run: npm install
+    - uses: actions/cache@v3
+      with:
+        key: ${{ runner.os }}-dprint-${{ hashFiles('package-lock.json', '.dprint.jsonc') }}
+        path: ~/.cache/dprint
+        restore-keys: |
+          ${{ runner.os }}-dprint-
+    - run: npx dprint fmt
+    - uses: stefanzweifel/git-auto-commit-action@v4.16.0
+      with:
+        # TODO: What to use for these?
```

Comment:
> TODO: What to use for these?

we are setting up committer info that way in other place:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/.github/workflows/UpdateCodeowners.yml#L30-L31

Reply:
Perfect, thanks!